internal/provider/resources/install/ssh/gcp.go

```diff
@@ -26,14 +28,18 @@ type sshGcpIamWrite struct {
 }
 
 type sshGcpIamWriteModel struct {
-	ProjectId types.String `tfsdk:"project_id" json:"projectId,omitempty"`
-	State     types.String `tfsdk:"state" json:"state,omitempty"`
-	Label     types.String `tfsdk:"label" json:"label,omitempty"`
+	GroupKey      types.String `tfsdk:"group_key" json:"groupKey,omitempty"`
+	IsSudoEnabled types.Bool   `tfsdk:"is_sudo_enabled" json:"isSudoEnabled,omitempty"`
+	Label         types.String `tfsdk:"label" json:"label,omitempty"`
+	ProjectId     types.String `tfsdk:"project_id" json:"projectId,omitempty"`
+	State         types.String `tfsdk:"state" json:"state,omitempty"`
 }
 
 type sshGcpIamWriteJson struct {
-	State string  `json:"state"`
-	Label *string `json:"label,omitempty"`
+	GroupKey      *string `json:"groupKey"`
```

Comment:
ProjectId is not required here?

Reply:
No this represents the body of the request, and the ProjectID isn't included. Instead it's returned by `func (r *sshGcpIamWrite) getId(data any) *string`